Validation/src/ValKalSeed.cc

```diff
@@ -77,8 +79,8 @@ namespace mu2e {
     return 0;
   }
 
-  int ValKalSeed::fill(const KalSeedCollection& coll,
-      art::Event const& event) {
+  int ValKalSeed::fill(const KalSeedCollection& coll, art::Event const& event) {
+    auto const& ptable = GlobalConstantsHandle<ParticleDataList>();
```

Comment:
A question for a possible followup PR.

I suggest making ptable a datamember of type `ParticleDataList const&` and initializing it in the c'tor, as in:

https://github.com/kutschke/Offline/blob/art_v3_15_00_prep/Mu2eBTrk/inc/ParticleInfo.hh
https://github.com/kutschke/Offline/blob/art_v3_15_00_prep/Mu2eBTrk/src/ParticleInfo.cc

This works because information from the GlobalConstantsService does not change over the job.  Getting the handle every event is right for Proditions objects, which may change over the course of a job.

The trivial speed improvement is not the issue - it's providing a model of doing things correctly.

Reply:
If you want to address this tonight, let me know and I will watch for it before I merge.